lib/govuk_publishing_template.rb

```diff
@@ -0,0 +1,8 @@
+require_relative "govuk_admin_template"
+
+class GovukPublishingTemplate < GovukAdminTemplate
+  def apply
+    add_sidekiq
```

Comment:
Yes, it should, thanks.  I've updated the commit and tested manually.

I've also added two more commits so if you could check those that'd be great.  Otherwise good to merge.